tensorboard/plugins/interactive_inference/tf_interactive_inference_dashboard/tf-interactive-inference-dashboard.html

```diff
@@ -4387,6 +4390,9 @@ <h2>Show similarity to selected datapoint</h2>
             // show toast with the error
             this.showToast_(result.value.error);
             this.$.spinner.hidden = true;
+            if (errorFn != null) {
```

Comment:
may want to be careful with this -- if you change this to `!==`, it will start to fail.
```
> null == undefined
< true
> null === undefined
< false
```

How about default arg to errorFn in L4384? (`errorFn= () => {}`)

Reply:
added default arg. kept this null check  in place. we use the == and != syntax around WIT